<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;supprimer&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>AJOUTER</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1141</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>saisir données du chambre</string>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1151</width>
         <height>521</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    background: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:1, y2:1, 
        stop:0 #CEBEA7,   
        stop:0.25 #8F959A,
        stop:0.5 #4B667B,  
        stop:0.75 #3B4672, 
        stop:1 #22223B     
    );
    color: #F2E9E4;
    border: 2px solid #22223B;
    border-radius: 10px; 
    margin-top: 15px;
}</string>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLineEdit" name="nb_c">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>140</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="type_c">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>200</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deluxe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>suite</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>210</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Type de chambre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="capacite">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>280</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="status_d">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>370</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Disponible </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Non disponible</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>200</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status de nettoyage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>140</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Services inclus</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Numéro de la chambre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>290</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>capacite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status de disponibilité</string>
        </property>
       </widget>
       <widget class="QComboBox" name="service">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>140</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>TV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WIFI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MINIBAR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coffre fort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Machine à café/thé</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="status_n">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>200</y>
          <width>81</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>nettoye</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Non nettoye</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="annuler">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>410</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>ANNULER</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouter">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>340</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>AJOUTER</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AFFICHER</string>
     </attribute>
     <widget class="QTableView" name="aff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1191</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="supp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Trie">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Trie</string>
      </property>
     </widget>
     <widget class="QComboBox" name="tri">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>330</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>numero</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="valeur">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>330</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Rechercher">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>370</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>400</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QComboBox" name="rech">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>330</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>id</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>numero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>service</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="verifier_chambre">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>20</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Verifer Chambres</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>MODIFIER</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1141</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>nouvelles données du chambre</string>
      </property>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>350</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>MODIFIER</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1141</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    background: qlineargradient(
        spread:pad, 
        x1:0, y1:0, x2:1, y2:1, 
        stop:0 #CEBEA7,   
        stop:0.25 #8F959A,
        stop:0.5 #4B667B,  
        stop:0.75 #3B4672, 
        stop:1 #22223B     
    );
    color: #F2E9E4;
    border: 2px solid #22223B;
    border-radius: 10px; 
    margin-top: 15px;
}</string>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLineEdit" name="nb_c2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>150</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="type_c2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>210</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>standard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>deluxe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>suite</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="capacite_2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>290</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="status_d2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>380</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Disponible </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Non disponible</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="service2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>160</y>
          <width>101</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>TV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WIFI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MINIBAR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coffre fort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Machine à café/thé</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="status_n2">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>230</y>
          <width>91</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>nettoye</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Non nettoye</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>230</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status de nettoyage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>160</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Services inclus</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID chambre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Numero de la chambre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Type de chambre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>300</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Capacité</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>390</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Status de disponibilité</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="id_m">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="recuperer">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Récuperer</string>
        </property>
       </widget>
      </widget>
      <zorder>groupBox_3</zorder>
      <zorder>modifier</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>STATISTIQUE</string>
     </attribute>
     <widget class="QGraphicsView" name="stat">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>1131</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
